dora-parser/src/ast/dump.rs

```diff
@@ -624,15 +624,15 @@ impl<'a> AstDumper<'a> {
     }
 
     fn dump_expr_dot(&mut self, expr: &ExprDotType) {
-        self.indent(|d| d.dump_expr(&expr.rhs));
         dump!(self, "dot @ {} {}", expr.pos, expr.id);
         self.indent(|d| d.dump_expr(&expr.lhs));
+        self.indent(|d| d.dump_expr(&expr.rhs));
```

Comment:
Why this change? This should be the same as e.g. in dump_expr_bin.

Reply:
I found it confusing that we were printing the rhs before the lhs ... I have no issue changing it back.
What was the reasoning doing it this way?

Reply:
No particular reason, I have an easier time understanding the tree if I pretend to rotate the screen to the right: then the lhs is left, the operator is in the middle and rhs is on the right. We can change this but then all uses need to change. Either way it doesn't belong in this PR.

Reply:
Agreed, will remove it from this PR!